__mocks__/mockCreators/uss.ts

```diff
@@ -61,3 +62,16 @@ export function createFavoriteUSSNode(session, profile) {
     ussNodeF.tooltip = "/u/myuser/usstest";
     return ussNodeF;
 }
+
+export function addSessionNode(theTree, theSession, theProfile) {
```

Comment:
At some point we should revisit why we need these complicated mocks. I think that it is a sign that we are not doing something right.

Reply:
I fixed this :3 Was just being lazy about removing it at 1:30 am the other day